### PR TITLE
Content migration from changelogs to release notes

### PR DESCRIPTION
**Pull request by rachelwhitton**
- [x] Backfill all entries from changelogs to release notes 
- [x] Assign categories throughout
- [x] Peer review
- [x] Delete /changelogs from the Gatsby site, including template files

Post Launch: 
- [ ] Add redirect from /changelog* to /release-notes in AGCDN

### CODE REVIEW

**Comment by rachelwhitton** [comment]
@IngridKwok here's where we can finish the content migration work. I'm still adding entries older than Jan 2022, but in the meantime if you have bandwidth you can start working on categories in this same branch/PR

File: source/releasenotes/2015-05-01-webinars.md
Comment (IngridKwok) [inline source/releasenotes/2015-05-01-webinars.md:1]: @rachelwhitton Noting we do not have a tag for "training/webinars".

Comment (rachelwhitton) [reply source/releasenotes/2015-05-01-webinars.md:1]: @IngridKwok I would prefer we use the events category in favor of adding a new category

**Comment by rachelwhitton** [comment]
@stevector can I get a peer review from you on my work in this PR specific to the removal of changelogs from the codebase? 

Here's where I deleted changelogs: https://github.com/pantheon-systems/documentation/pull/8851/commits/598b9c92d3981c4abc660ef75377f6f4b50da641 

Here's where I updated the query for our home page so that recent release notes are shown instead of changelogs: https://github.com/pantheon-systems/documentation/pull/8851/commits/581d8c3e585f0dbd7dced8c7e79f0a1c7f1ca1db 

Your peer review on category assignments across all backfill entries is most welcomed, but not required. Ingrid and I plan to peer review those together

**Comment by stevector** [comment]
@rachelwhitton Looks good. I was wondering if a giant scrolling list of all release notes ever would be unusable huge. While it would still make sense to add back the date based pagination, this page is more usable than I expected: https://pr-8851-documentation.appa.pantheon.site/release-notes/

**Comment by rachelwhitton** [comment]
+1 from me on this, all categories have been peer-reviewed on my side. Final todo is confirm peer review is complete from @IngridKwok then we can deploy this PR and add the redirect

**Comment by IngridKwok** [comment]
@rachelwhitton We are ready to deploy this. Thank you!

**Pull request merged**